csharp/private/repositories.bzl

```diff
@@ -98,6 +98,391 @@ def csharp_repositories():
         sha256 = "912f6eed993e77c83cb8b92db72d50df7d06e4c4a02486474eae460728291989",
     )
 
+	nuget_package(
+		name = "netstandard2plus"
+		package = "NetStandard.Library"
```

Comment:
This has no dependencies for netstandard2.0, including everything inline in `build/netstandard2.0/ref`. It also contains this targets file:

```xml
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup>
    <NETStandardLibraryPackageVersion>2.0.3</NETStandardLibraryPackageVersion>
  </PropertyGroup>

  <!-- Only add references if we're actually targeting .NETStandard.
       If the project is targeting some other TFM that is compatible with NETStandard we expect
       that framework to provide all references for NETStandard, mscorlib, System.* in their own 
       targeting pack / SDK.
       Choose/When is required to avoid an issue where the old desktop project system in VS  
       will evaluate this item ignoring the condition in order to populate the references. --> 
  <Choose>
    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard'">
      <ItemGroup>
        <Reference Include="$(MSBuildThisFileDirectory)ref\netstandard.dll">
          <!-- Private = false to make these reference only --> 
          <Private>false</Private>
          <!-- hide these from Assemblies view in Solution Explorer, they will be shown under packages --> 
          <Visible>false</Visible>
          <NuGetPackageId>NETStandard.Library</NuGetPackageId>
          <NuGetPackageVersion>$(NETStandardLibraryPackageVersion)</NuGetPackageVersion>    
        </Reference>
```